src/tests/ftest/server/metadata.yaml

```diff
@@ -9,7 +9,7 @@ hosts:
     - server-D
   test_clients:
     - client-C
-timeout: 300
+timeout: 900
```

Comment:
Is this test going over 300 in normal circumstance ?

Reply:
The test_metadata_fillup() is shorter and doesn't require increasing the timeout. The test_metadata_addremove() is intentionally lengthy and so an increase is needed to make sure if/when it is run (I did most recently with this PR) that it will not time out. Originally timeout had been 1800 seconds. I reduced it in some recent commits to ~300 seconds but probably didn't consider the addremove test (or it may not have been really able to run anyway, since we have been focusing on improving code behavior as we approach and get to an out of space situation - I forget specifically why I reduced it in the earlier commits).

These tests are usually run manually by me when testing rdb and primarily the shorter test_metadata_fillup(). They are tagged full_regression so would potentially be part of a weekly or other periodic run but not on each PR.